webui/react/src/pages/WorkspaceList.tsx

```diff
@@ -273,7 +272,8 @@ const WorkspaceList: React.FC = () => {
     workspaces ]);
 
   useEffect(() => {
-    fetchWorkspaces();
+    setIsLoading(true);
+    fetchWorkspaces().then(() => setIsLoading(false));
```

Comment:
good catch, probably best to remove it from the `fetchWorkspaces` instead though? since if there is already a polling request in flight when user updates filters, when that resolves it will erroneously set `isLoading` to false.

Reply:
Sure, that should work